pymatgen/analysis/phase_diagram.py

```diff
@@ -260,71 +257,52 @@ def from_dict(cls, d):
         return cls(d["composition"], entry)
 
 
-class PhaseDiagram(MSONable):
+class DBPhaseDiagram(MSONable):
```

Comment:
Change to PhaseDiagram, Now inherits from DBPhaseDiagram which has the simpler constructor and the stander as_dict and from_dict